modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
+
+.Supported versions of {oadp-short}
+[width="100%",cols="10%,12%,12%,13%,13%,20%,20%, options="header",]
+|===
+
+|Version
+|{OCP-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+|Extended Update (EUS)
+|Extended Update Support Term 2 (EUS Term 2)
+
+
+
+|1.4
+a|
+* 4.14
+* 4.15
+* 4.16
+|10 Jul 2024
+|Release of 1.5
+|Release of 1.6
+a|
+27 Jun 2026
+
+(EU must be on {OCP-short} 4.16)
+a|
+27 Jun 2027
+
+(EUS Term 2 must be on {OCP-short} 4.16)
+
+|1.3
+a|
+* 4.12
+* 4.13
+* 4.14
+* 4.15
+|29 Nov 2023
+|10 Jul 2024
+|Release of 1.5
+a|
+31 Oct 2025
+
+(EUS must be on {OCP-short} 4.14)
```

Comment:
Global comment:
OCP-version-specific information can be handled as part of manual cherry-picks into the relevant release branches. In this case, it's 4.14:
https://github.com/openshift/openshift-docs/tree/enterprise-4.14